src/components/Whiteboard/ElementBehaviors/Resizable/ResizeElement.tsx

```diff
@@ -16,39 +16,41 @@
 
 import { Dispatch, useCallback, useState } from 'react';
 import { useUnmount } from 'react-use';
-import { Point, useWhiteboardSlideInstance } from '../../../../state';
-import { calculateBoundingRectForElements } from '../../../../state/crdt/documents/elements';
+import {
+  calculateBoundingRectForElements,
+  useWhiteboardSlideInstance,
+} from '../../../../state';
 import {
   useElementOverride,
   useSetElementOverride,
 } from '../../../ElementOverridesProvider';
 import { useLayoutState } from '../../../Layout';
+import { getRenderProperties } from '../../../elements/line/getRenderProperties';
 import { useSvgCanvasContext } from '../../SvgCanvas';
 import { gridCellSize } from '../../constants';
 import { DragEvent, ResizeHandle } from './ResizeHandle';
-import { Dimensions, ResizeHandlePosition } from './types';
+import {
+  Dimensions,
+  LineElementProps,
+  PolylineAndShapeElementsProps,
+} from './types';
 import { calculateDimensions } from './utils';
 
 export type ResizeHandleWrapperProps = {
-  handlePosition: ResizeHandlePosition;
   startDimensions: Dimensions | undefined;
-  containerWidth: number;
-  containerHeight: number;
   forceLockAspectRatio?: boolean;
   onDrag: Dispatch<Dimensions>;
   onDragStart: Dispatch<DragEvent>;
   onDragStop: Dispatch<DragEvent>;
-};
+} & (LineElementProps | PolylineAndShapeElementsProps);
```

Comment:
I suggest to add a separate prop: `params` (or `resizeParams` or some better name)  of the type: `LineElementProps | PolylineAndShapeElementsProps`